scripts/run-ginkgo-integration-suite.sh

```diff
@@ -24,6 +24,15 @@ function load_test_parameters(){
   if [[ ! -z $TARGET_ADDON_VERSION ]]; then
     EXTRA_OPTIONS+=" --target-addon-version $TARGET_ADDON_VERSION"
   fi
+
+  if [[ ! -z $INITIAL_MANIFEST_FILE ]]; then
+    EXTRA_OPTIONS+=" --initial-manifest-file $INITIAL_MANIFEST_FILE"
+  fi
+
+  if [[ ! -z $TARGET_MANIFEST_FILE ]]; then
```

Comment:
Question: are these two parameters exclusive? Like can they both be set?

Reply:
Not really. I didn't want to complicate the logic here since we already have that check in the test. The test checks if we have both the parameters defined and not empty.